### PR TITLE
add astral-manga connector

### PR DESCRIPTION
**Pull request by Neths**

### CODE REVIEW

**Comment by MikeZeDev** [comment]
Need to provide icon

File: src/web/mjs/connectors/AstralManga.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class AstralManga extends WordPressMadara {
+
+    constructor() {
+        super();
+        super.id = 'astralmanga';
+        super.label = 'Astral-manga';
+        this.tags = [ 'manga', 'webtoons', 'french' ];
+        this.url = 'https://astral-manga.fr/';
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/AstralManga.mjs]: Remove leading /

**Review by MikeZeDev — CHANGES_REQUESTED** [review]
Minor changes required
* add icon
* Fix url

**Comment by Neths** [comment]
minor changes fixed

**Review by MikeZeDev — APPROVED** [review]

**Comment by ronny1982** [comment]
[Missing in HaruNeko](https://github.com/search?q=repo%3Amanga-download%2Fharuneko%20%27astralmanga%27&type=code)

**Comment by MikeZeDev** [comment]
Added to Haruneko https://github.com/manga-download/haruneko/pull/344

**Pull request merged**